java/java-common/src/main/java/org/cloudburstmc/protocol/java/packet/play/ContainerSetDataPacket.java

```diff
@@ -0,0 +1,30 @@
+package org.cloudburstmc.protocol.java.packet.play;
+
+import lombok.*;
+import org.cloudburstmc.protocol.java.JavaPacket;
+import org.cloudburstmc.protocol.java.data.window.property.WindowProperty;
+import org.cloudburstmc.protocol.java.handler.JavaPlayPacketHandler;
+import org.cloudburstmc.protocol.java.packet.type.JavaPlayPacketType;
+
+@Data
+@EqualsAndHashCode(doNotUseGetters = true, callSuper = false)
+@AllArgsConstructor
+public class ContainerSetDataPacket extends JavaPacket<JavaPlayPacketHandler> {
+    private int windowId;
+    private int rawProperty;
```

Comment:
Should be replaced with a property enum as mentioned on wiki.vg: https://wiki.vg/Protocol#Window_Items